xtask/src/templates/control_id_zkr.rs

```diff
@@ -18,3 +18,11 @@ pub const ALLOWED_IDS_ROOT: &str = "{}";
 pub const RECURSION_CONTROL_IDS: [(&str, &str); {}] = [{}];
 
 pub const BN254_CONTROL_ID: &str = "{}";
+
+#[test]
+fn stable_root() {{
+    // This tests that none of the control IDs have changed unexpectedly
+    // If you have _intentionally_ changed control IDs, update this hash and bootstrap.
+
+    assert_eq!(ALLOWED_IDS_ROOT, "88c1f749250aba181168c33839d7a351671e7a5b7f3e746dde91ef6c6e9ef344");
+}}
```

Comment:
I don't think a test should go into a template. Could we move the test into a place that isn't generated by an xtask?

Reply:
That makes sense, I moved it to `risc0/zkvm/src/host/recursion/tests.rs`